examples/js_of_ocaml/readme.md

```diff
@@ -0,0 +1,35 @@
+# Js_of_ocaml TodoMVC Example
+
+> Js_of_ocaml is a compiler of OCaml bytecode to Javascript.
+
+## Resources
+
+- [Website](http://ocsigen.org/js_of_ocaml/)
+- [GitHub](https://github.com/ocsigen/js_of_ocaml)
+- [Try Js_of_ocaml](http://try.ocamlpro.com/js_of_ocaml/)
+
+### Support
+
+Js_of_ocaml is part of [Ocsigen project](http://ocsigen.org/).
+
+- [Mailing list](https://sympa.inria.fr/sympa/subscribe/ocsigen)
+- IRC : #ocsigen on irc.freenode.net
+
+
+## Implementation
+
+Open `index.html` in your browser to try the application.
+
+If you want to build the application on your own:
+
+1. Install Js_of_ocaml. An easy way is to use [opam](https://opam.ocaml.org/).
+2. See the `build.sh` script.
+
+
+## Credit
+
+Created by [Stéphane Legrand](https://stephanelegrand.wordpress.com/).
```

Comment:
Try to be consistent. Here you're linking to your personal site while for other people you link to their GitHub profile :)